packages/cli-lib/templates.js

```diff
@@ -15,24 +13,18 @@ const ANNOTATION_KEYS = {
   screenshotPath: 'screenshotPath',
 };
 
-const getFileAnnotations = filePath => {
-  try {
-    const data = fs.readFileSync(filePath, 'utf8');
-    const match = data.match(ANNOTATIONS_REGEX);
-    const annotation = match && match[1] ? match[1] : '';
-    return annotation;
-  } catch (err) {
-    logger.debug(err);
-    return '';
-  }
-};
-
 const getAnnotationValue = (annotations, key) => {
   const valueRegex = new RegExp(`${key}${ANNOTATION_VALUE_REGEX}`);
   const match = annotations.match(valueRegex);
   return match ? match[1].trim() : null;
 };
 
+const buildAnnotationValueGetter = source => {
+  const match = source.match(ANNOTATIONS_REGEX);
+  const annotation = match && match[1] ? match[1] : '';
+  return annotationKey => getAnnotationValue(annotation, annotationKey);
+};
+
```

Comment:
Works as expected here https://github.com/HubSpot/hubspot-cms-vscode/pull/117/files#diff-dfcd8afcb372a75a1eb1bdc0af57d36a5e0a727dbca99350779abb21333a298fR65

and also tested the `marketplace-validate` command out and that works as expected

Reply:
Cool! This pattern looks cleaner 🙌